docs/_api/decoratorcontroller.md

```diff
@@ -229,17 +229,29 @@ Metacontroller requires you to be explicit about the version you expect
 because it does conversion for you as needed, so your hook doesn't need
 to know how to convert between different versions of a given resource.
 
-Within each attachment type (e.g. in `attachments['Pod.v1']`),
-there is another associative array that maps from the attachment's
-`metadata.name` to the JSON representation, like what you might get
-from `kubectl get <attachment-resource> <attachment-name> -o json`.
-
-For example, a Pod named `my-pod` could be accessed as:
+Within each child type (e.g. in `children['Pod.v1']`),
+there is another associative array that maps from the child's path relative
+to the parent to the JSON representation. If the parent and child are of the
+same scope - both cluser or both namespaced - then the key is simply the
+object's `.metadata.name`,  like what you might get from
+`kubectl get <child-resource> <child-name> -o json`. If the parent is
+cluster scoped and the child is namespaced scope then the key will be of the
+form: `{.metadata.namespace}/{.metadata.name}`. This is to disambiguate between
+two children with the same name in different namespaces.
+
+For example, a Pod named `my-pod` in the `default` namespace could be accessed
+as:
 
 ```js
 request.attachments['Pod.v1']['my-pod']
 ```
 
+or if the parent resource is cluster scoped as:
+
+```js
+request.attachments['Pod.v1']['default/my-pod']
+`
```

Comment:
Just noticed in the [deploy preview](https://deploy-preview-71--metacontroller.netlify.com/api/decoratorcontroller/) that this is missing a couple backticks. Looks like the one in CompositeController is fine though.